components/Layout/index.tsx

```diff
@@ -132,20 +129,20 @@ export const Layout: React.VFC<{
               logoAltText: tsln.logoAltText,
             }}
           />
-          <Heading
+          <h1
             id="applicationTitle"
-            title={title}
-            className="mb-8 mt-4 sm:mt-12 sm:w-[100%]"
-          />
+            className="font-header-gc font-[700] text-[#333333] text-[38px] leading-[42px] mb-8 mt-4 sm:mt-12 sm:w-[100%]"
```

Comment:
globals.css has a class  .h1  aren't we duplicating code?  is a question

Reply:
Questions page and ErrorsSummary also use the message component, maybe added them to this PR